src/Database/Redis/Schema.hs

```diff
@@ -133,13 +133,13 @@ data RedisException
   | TransactionAborted
   | TransactionError String
   | CouldNotDecodeValue (Maybe ByteString)
-  | LockAcquireTimeout
   | UnexpectedStatus String Hedis.Status
   | EmptyAlternative  -- for 'instance Alternative Tx'
   deriving (Show, Exception)
 
 -- | Time-To-Live for Redis values. The Num instance works in (integral) seconds.
 newtype TTL = TTLSec { ttlToSeconds :: Integer }
+  deriving (Show)
```

Comment:
Cosmetics: I like to add explicit deriving strategies everywhere on newtypes, especially on when there are lines with explicit strategies around. I'm never sure what the default strategy is, and I _think_ it depends on compiler flags.
```suggestion
  deriving stock (Show)
```